src/player.js

```diff
@@ -3,38 +3,88 @@ class Character {
         this.name = name;
         this.hp = 100;
         this._speed = 100;
+        this._isMoving = false;
```

Comment:
Do we need underscore sign between some variables names? There is no consistency now.